### DISCUSSION TITLE
V2 Admin translation

### DISCUSSION THREAD

**Discussion by tirocoder** [medusajs/medusa#9920] [Feature Requests]
Can we add our own language file to our starter project? What are the methods for this? I'm asking so I can try it in my own local starter repo. I don't want to wait for it to be added to the medusa repo.

**Comment by SalahAdDin** [comment]
The admin should include many languages, if you don't find yours, you could open a PR for helping to translate it.

**Reply by tirocoder** [reply]
I'm asking so I can try it in my own local starter repo. I don't want to wait for it to be added to the medusa repo

**Reply by SalahAdDin** [reply]
You can make it on your local and open a new PR to contribute to Medusa.
